WiserTaskScheduler/AutoUpdater/Services/UpdateService.cs

```diff
@@ -147,8 +149,11 @@ private void UpdateWts(WtsModel wts, List<VersionModel> versionList)
                 logger.LogInformation($"WTS '{wts.ServiceName}' is up-to-date.");
                 return;
             case UpdateStates.BreakingChanges:
-                logger.LogWarning($"Could not update WTS '{wts.ServiceName}' to version {versionList[0].Version} due to breaking changes since the current version of the WTS ({version}).{Environment.NewLine}Please check the release logs and resolve the breaking changes before manually updating the WTS.");
-                EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Manual action required", $"Could not update WTS '{wts.ServiceName}' to version {versionList[0].Version} due to breaking changes since the current version of the WTS ({version}).<br/>Please check the release logs and resolve the breaking changes before manually updating the WTS.", wts.ServiceName);
+                var subject = "WTS Auto Updater - Manual action required";
+                var message= $"Could not update WTS '{wts.ServiceName}' to version {versionList[0].Version} due to breaking changes since the current version of the WTS ({version}).{Environment.NewLine}Please check the release logs and resolve the breaking changes before manually updating the WTS.";
+                
+                logger.LogWarning(message);
+                InformPeople(wts,subject,message);
```

Comment:
I'm not going to reject the PR for this, but please separate parameters with spaces.